### PR TITLE
ASUIS-760: Webspark Webdir module shows wrong affiliations in Layout Builder interface

### PR DESCRIPTION
**Pull request by tbutterf**
This PR resolves ASUIS-760 as well as ASUIS-799 (In people Component type, clicking on parent does not add to list). It also adds parent/child relationships for when multiple units are selected and people exist in more than one unit.
See:
https://asudev.jira.com/browse/ASUIS-760
https://asudev.jira.com/browse/ASUIS-799

### CODE REVIEW

File: jquery-plugin/jstree/dist/jstree.min.js [hunk 1]
```diff
@@ -1,6 +1,2638 @@
...
\ No newline at end of file
+!function (a) {
```
Comment (mlsamuelson) [inline jquery-plugin/jstree/dist/jstree.min.js]: Please keep minified.

File: js/add-field.js [hunk 2]
```diff
@@ -6,21 +6,24 @@
...
-            default_values.push(data.node.id);
-            // Recreate tree with the default values.
-            update_tree(default_values);
+            if (data.node.id.includes(":")) {
+              // Get the existing values
+              default_values = $('.asurite-tree').val().split(',');
+              // Add the person to the tree.
+              default_values.push(data.node.id);
+              // Recreate tree with the default values.
+              update_tree();
```
Comment (mlsamuelson) [inline js/add-field.js]: Check that `default_values` isn't needed. If this wasn't intentional.

File: js/add-field.js [hunk 3]
```diff
@@ -45,63 +48,91 @@
...
-    Object.keys(temp).forEach(deptID => {
-      var sorted = temp[deptID];
-
-      const uniqueIds = new Set();
 
-      const unique = sorted.filter(element => {
-        const isDuplicate = uniqueIds.has(element.id);
-
-        uniqueIds.add(element.id);
+          if (departments.length > 1) {
```
Comment (mlsamuelson) [inline js/add-field.js:59]: Let's add a comment here to identify that this is what handles the nesting when multiple depts, in the UI.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

File: jquery-plugin/jstree/dist/jstree.min.js (hunk 1 above)
Comment (tbutterf) [reply jquery-plugin/jstree/dist/jstree.min.js]: Done.

File: js/add-field.js (hunk 2 above)
Comment (tbutterf) [reply js/add-field.js]: added back.

File: js/add-field.js (hunk 3 above)
Comment (tbutterf) [reply js/add-field.js:59]: comment added.

**Review by mlsamuelson — APPROVED** [review]
Thank you for the updates. Next person who has to debug an issue here will owe you one, @tbutterf . ;)

**Pull request merged**